interface/forms/newpatient/common.php

```diff
@@ -248,6 +248,29 @@ class="css_button link_submit" target='Main' onClick="top.restoreSession()">
 			</div>
 		</td>
      </tr>
+	<?php if($GLOBALS['set_pos_code_encounter']){ ?>
+        <tr>
+            <td><span class='bold' nowrap><?php xl('POS Code','e'); ?>: </span></td>
```

Comment:
use echo xlt and don't use the e parameter:
``````
<td><span class='bold' nowrap><?php echo xlt('POS Code'); ?>: </span></td>